benchmarks/dynamo/pr_time_benchmarks/benchmark_base.py

```diff
@@ -142,6 +219,8 @@ def append_results(self, path):
             for entry in self.results:
                 writer.writerow(entry)
 
+        self._write_to_json(os.path.dirname(os.path.abspath(path)))
```

Comment:
can we do that in collect_all, maybe writer after. 
```
    scribe_log_torch_benchmark_compile_time(
                name=self.name(),
                instruction_count=r,
            )
   ```
   write now this not called by default by all benchmarks. 
   I am thinking to also move the appending to CSV to be by default there in collect all.

Reply:
I think this need the path to write to, so I need to keep in `append_results`.  Maybe we could follow up with another PR to refactor both the CSV and JSON output to `collect_all`?